apps/docs-snippets2/src/contracts/managing-deployed-contracts.ts

```diff
@@ -0,0 +1,28 @@
+import { Contract, Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../env';
+import { EchoValues, EchoValuesFactory } from '../typegend';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+const abi = EchoValues.abi;
+const { waitForResult, contractId } = await EchoValuesFactory.deploy(wallet);
+
+await waitForResult();
+
+// #region with-contractId
+const deployedEchoContract = new Contract(contractId, abi, wallet);
+
+const { value: echoed10 } = await deployedEchoContract.functions
```

Comment:
Why renaming to `echoed10` here?

Reply:
Because I'm reusing the same file for two snippets: `with-contractId` and `with-b256`. Below this you'll find the other `with-b256` region where 50 is being echoed so I assigned it an `echoed50` variable name. Otherwise I'd have variable name conflicts. It didn't seem necessary to separate it out into two snippets, but I can do it if it's preferred.